src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+
+interface PersonProps {
+  person: Person;
+  getParent: (name: string) => Person | undefined;
+}
+export const PersonLink: React.FC<PersonProps> = ({ person, getParent }) => {
+  const { name, sex, born, died, fatherName, motherName, slug } = person;
+  const { slugs } = useParams();
+
+  const motherPerson = motherName ? getParent(motherName) : null;
+  const fatherPerson = fatherName ? getParent(fatherName) : null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === slugs,
+      })}
+    >
+      <td>
+        <Link
+          className={classNames({
+            'has-text-danger': sex === 'f',
+          })}
+          to={`/people/${slug}`}
+        >
+          {name}
+        </Link>
```

Comment:
create the <PersonLink person={person} /> and use it for all existing people;
<img width="652" alt="image" src="https://github.com/user-attachments/assets/e4f6ffe5-4de5-4833-999e-443bb6b49002">

And the same for mother and father